efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -27,6 +27,71 @@
 from keras import utils_keras
 
 
+class FuseFeatures(tf.keras.layers.Layer):
```

Comment:
What's meaning of change it to keras layers?

Reply:
When I loaded the model as keras.Model, I got the error `tenorflow.python.framework.errors_impl.FailedPreconditionError`
I was able to localize it before calling the `fuse_feature` function, so to fix this, I created a layer and started covering it with tests (this was the original reason).